com.avaloq.tools.ddk.check.ui.test/src/com/avaloq/tools/ddk/check/ui/test/quickfix/CheckQuickfixTest.xtend

```diff
@@ -84,5 +132,88 @@ class CheckQuickfixTest extends AbstractCheckQuickfixTest {
     assertQuickFixExistsAndSuccessfulInCustomerSource(IssueCodes::MISSING_ID_ON_CHECK,
       Messages.CheckQuickfixProvider_ADD_ID_LABEL, testSourceFileName, sourceContent, expectedContent)
   }
+
+  /**
+   * Test bulk-applying a quickfix.
+   * Tolerate up to ten timeouts, because occasionally new markers don't appear or not all markers are fixed;
+   * the problems appears to be in Eclipse.
+   */
+  @Test

```

Comment:
Why not mark it `@Flaky` then?

Reply:
Because @Flaky doesn't cause failing tests to be retried; it is only used to mark and filter tests.
But at @rmitin's suggestion, I have added (and used) a new @Retry annotation.